jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -113,6 +113,33 @@ byte coder() {
 		}
 	}
 
+/*[IF JAVA_SPEC_VERSION >= 16]*/
+	/**
+	 * Copy bytes from value starting at srcIndex into the bytes array starting at
+	 * destIndex. No range checking is needed. Caller ensures bytes is in UTF16.
+	 * 
+	 * @param bytes copy destination
+	 * @param srcIndex index into value
+	 * @param destIndex index into bytes
+	 * @param coder LATIN1 or UTF16
+	 * @param length the number of elements to copy
+	 * 
+	 * @return void
```

Comment:
I don't think this is appropriate: please remove.

Reply:
It has been removed.